### DISCUSSION TITLE
How to have gollum ignore a directory?

### DISCUSSION THREAD

**Discussion by sbeaumont** [gollum/gollum#1863] [Q&A]
Hi,

I've done my best to search the documentation, but can't find an option to have gollum ignore a directory. We have a k8s directory in the root of the repo that has the cloud config, and this folder shows up in overviews of directories (even though there's no .md files there). Is there a way to have gollum ignore this directory?

**Comment by benjaminwil** [comment]
This was discussed recently here: https://github.com/gollum/gollum/issues/1810

I think that the short answer, right now, is no. But it’s something I’d be interested in speccing out.

**Comment by dometto** [comment]
Two possible workarounds:

* have the wiki in a separate repository under the repo root and use the `--page-file-dir` option.
* since gollum doesn't yet support [submodules](https://github.com/gollum/gollum/issues/1712), you could try turning the k8 directory into a submodule, and gollum should be unable to see it :)
